src/api/app/models/event/build.rb

```diff
@@ -23,6 +23,11 @@ def custom_headers
 class Event::BuildSuccess < Event::Build
   self.raw_type = 'BUILD_SUCCESS'
   self.description = 'Package has succeeded building'
+  after_create :send_to_bus
```

Comment:
There are instances of `after_create` and `after_commit` in the Event classes, I think we decided on `after_commit` if i remember correctly?

Reply:
Hm, just checked the [previous PR](#3924) and there we use both... I'll switch to after_commit then.